v2/options.go

```diff
@@ -72,6 +73,14 @@ func ZeroLengthSectionAsEOF(enable bool) Option {
 	}
 }
 
+// WithSkipOffset sets the start offset we should seek to the in the traversal
+// when writing out a CAR.
```

Comment:
It would be great if the docs mention that the option only applies to selective/traversal writer

Reply:
@masih Done.